src/routes/business/example-routes.js

```diff
@@ -9,12 +9,22 @@ const exampleDalConnectionRoute = {
   method: 'GET',
   path: '/example',
   handler: async (_request, h) => {
-    const dal = await dalConnector(getSbiInfo, variables, email)
-    const dalData = dal.data
+    const response = await dalConnector(getSbiInfo, variables, email)
+    const responseData = response.data
```

Comment:
Be consistent with the use of this variable eg its used on line 27 but not on line 17.

i dont really think its needed and `response.data` is sufficient